sinfo25/assets/css/style.css

```diff
@@ -6021,13 +5967,15 @@ div.clear input.btn:hover {
 }
 .hacky .photo-wrapper:hover img,
 .member .photo-wrapper:hover img {
-  -webkit-transform:scale(1.1); /* Safari and Chrome */
-  -moz-transform:scale(1.1); /* Firefox */
-  -ms-transform:scale(1.1); /* IE 9 */
-  -o-transform:scale(1.1); /* Opera */
-  transform:scale(1.1);
+  /*-webkit-transform:scale(1.1); /* Safari and Chrome */

```

Comment:
Should this stay here as a comment or may we just remove it?

Reply:
I did think about it, but if it's causing trouble might as well remove it.
The fact that I just commented the web-kit and not all of them was a distraction.